ctapipe/image/modifications.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+from numba import njit
+from ..core.component import TelescopeComponent
+from ..core.traits import FloatTelescopeParameter, BoolTelescopeParameter, Int
+from ..instrument import SubarrayDescription
+
+
+def add_noise(image, noise_level, rng=None, correct_bias=True):
+    """
+    Create a new image with added poissonian noise
+    """
+    if not rng:
+        rng = np.random.default_rng()
+    noisy_image = image.copy()
+    noise = rng.poisson(noise_level)
```

Comment:
If `noise_level` is a single number (it isn't when used thorugh the Component, but could be if something uses it on its own) each pixel get's the same value added.

Reply:
done